packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -257,7 +258,9 @@ export function createOdspNetworkError(
                 errorMessage, { canRetry: true, retryAfterMs }, driverProps);
             break;
     }
-
+    if (innerMostErrorCode === "fluidInvalidSchema") {
```

Comment:
All of the other changes can go into main. But I believe these 3 lines can't, as it will create a new error type that clients need to use. Some of our clients use dynamically loaded driver, so they may have newer version of the driver (that creates these errors) with older version of Loader & Host app, not knowing anything about this error.

I'd get all the changes other than these 3 lines into main and add breaking.md mentioning about this new error type and how we expect host to deal with it.
And submit these 3 lines to next branch, after your original change propagates there.